client/src/components/DatePicker/DatePicker.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react'
+import './DatePicker.css'
+import imageFile from '../../images/Search Icon_outlined.svg'
+
+function handleClick(e) {
+  e.preventDefault();
+  alert("You clicked the button");
+}
+
+const DatePicker = (props) => 
+<>
+<label for='requestedDate'>Requested Date*</label>
```

Comment:
Can you make the label text here a prop that someone can pass in?

Reply:
Done